opencti-platform/opencti-graphql/package.json

```diff
@@ -27,6 +27,7 @@
     "node": ">= 16.*"
   },
   "dependencies": {
+    "@aws-sdk/credential-providers": "^3.110.0",
```

Comment:
Currently we decide to fix all the versions you depends on. Maybe we will change that in the future but please remove the ^ for now